test/app.test.js

```diff
@@ -475,7 +475,7 @@ describe("app", function() {
                     .input('31')
                     .check.interaction({
                         state: 'No_SA_ID_Year_Entry',
-                        reply: /Please enter the year the patient was born. For example: 1982/
+                        reply: /The date of birth entered was incorrect. /
```

Comment:
I think the test is supposed to check all the text that would be displayed on screen. Please change this to be include the 'Please enter the year...' text as well and let me know what the tests say.

Reply:
It looks like this is the usual way to test this stuff so :+1: